code/game/objects/items/pet_carrier.dm

```diff
@@ -146,6 +167,8 @@
 	cut_overlay("locked")
 	if(open)
 		icon_state = initial(icon_state)
+	else if(name == "Xenological Containment Unit")
```

Comment:
Override the proc for the xenobio subtype instead of snowflake name based shit that breaks if you use a labeller on it

Reply:
implying I know how to override a proc

Reply:
I fixed it, now it checks the item instead of the name